sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,222 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 #### Tasks
 
-- [ ] Create the component `ArtPieceDetails`
-- [ ] `ArtPieceDetails` props: `image`, `title`, `artist`, `year`, `genre`
-- [ ] Create the page `pages/art-pieces/[slug]` that renders `ArtPieceDetails`
-- [ ] Read the query parameter `slug` from `next/router`
-- [ ] Use the `slug` to find the art piece to display
+- [ ] Create a new component for displaying Art Piece Details.
+- [ ] Create a single page that renders the Art Piece Details, displaying information for the selected art piece.
+- [ ] Write tests for the Art Piece Details component to confirm that all information is rendered correctly.
+
+### 5. Favorite Page
+
+The Favorite page will display a list of all favorited art pieces. Each art piece will show its image, artist’s name, title, and a favorite button.
 
----
+Users should be able to save their favorite art pieces both on the Gallery page and the Detail page.
 
-### User Story 5: Favorites
+To achieve this, we’ll divide the tasks into two parts:
+
+1. Implement the favorite button where needed.
+2. Create the Favorite page.
+
+### 5.1. Favorite Button
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** mark art piece pieces as favorites
+**I want to** mark art pieces as favorites
 
-**so that** I can find them easier.
+**so that** I can find them more easily.
 
 #### Acceptance Criteria
 
-- [ ] The favorite-button is displayed in each entry in the list view
-- [ ] The favorite-button is displayed in the details view
-- [ ] The favorite-button is NOT displayed in the spotlight view
-- [ ] Clicking the favorite-button on a non-favorite piece saves it as a favorite
-- [ ] Clicking the favorite-button on a favorite piece removes it from favorites
+- [ ] The favorite button is displayed on each entry in the list view.
+- [ ] The favorite button is displayed in the details view.
+- [ ] The favorite button is **NOT** displayed in the spotlight view.
+- [ ] Clicking the favorite button on a non-favorite piece saves it as a favorite.
+- [ ] Clicking the favorite button on a favorited piece removes it from favorites.
 
 #### Tasks
 
-- [ ] Create an additional state `artPiecesInfo` to save further information for art pieces
-- [ ] Make the `artPiecesInfo` state globally available for all pages
-- [ ] Store the favorite flag in the `artPiecesInfo`
-- [ ] Reference art pieces by `slug` in the additional state
-- [ ] Create the component `FavoriteButton`
-- [ ] `FavoriteButton` props: `isFavorite`, `onToggleFavorite`
-- [ ] Render the `FavoriteButton` component in the `ArtPiecePreview` and `ArtPieceDetails` component
-
----
+- [ ] Create a new component for the favorite button.
+- [ ] Store this additional information (favorite status) in a state. We will also be adding comments to each art piece, so consider how to store both types of information together.
+- [ ] Write tests to verify that the favorite button appears in the list view and details view but is not shown in the spotlight.
```

Comment:
I think we should give the students a bit more guidance here on how to check whether a certain art piece is favorite or not. Maybe something like "Create a function for checking whether an art Piece is included in the favoriteIds list".

Reply:
Added it as a hint.